packages/core/handle/tests/handle-specs.js

```diff
@@ -567,4 +568,106 @@ describe('handle', () => {
 			expect(actual).toEqual(expected);
 		});
 	});
+
+	describe('#forwardCustomWithPrevent', () => {
+		test('should pass an object with `type` when no adapter is provided', () => {
```

Comment:
We are checking the `preventDefault` function too. Please add a description for this.

Reply:
✅ I added a description for the next test also. 😉